src/containers/Tenant/Diagnostics/Describe/Describe.tsx

```diff
@@ -0,0 +1,95 @@
+import {useCallback} from 'react';
+import {useDispatch} from 'react-redux';
+import cn from 'bem-cn-lite';
+// @ts-ignore
+import JSONTree from 'react-json-inspector';
+import 'react-json-inspector/json-inspector.css';
+
+import {Loader} from '@gravity-ui/uikit';
+
+import {prepareQueryError} from '../../../../utils/query';
+import {useAutofetcher, useTypedSelector} from '../../../../utils/hooks';
+import {
+    getDescribe,
+    resetLoadingState,
+    setCurrentDescribePath,
+} from '../../../../store/reducers/describe';
+
+import './Describe.scss';
+
+const b = cn('kv-describe');
+
+const expandMap = new Map();
+
+interface IDescribeProps {
+    tenant: string;
+}
+
+const Describe = ({tenant}: IDescribeProps) => {
+    const dispatch = useDispatch();
+
+    const {currentDescribe, error, loading, wasLoaded} = useTypedSelector(
+        (state) => state.describe,
+    );
+
+    const {autorefresh, currentSchemaPath} = useTypedSelector((state) => state.schema);
+
+    const fetchData = useCallback(
+        (isBackground: boolean) => {
+            if (!isBackground) {
+                dispatch(resetLoadingState());
+            }
+
+            const path = currentSchemaPath || tenant;
+
+            dispatch(setCurrentDescribePath(path));
+            dispatch(getDescribe({path}));
+        },
+        [currentSchemaPath, tenant, dispatch],
+    );
+
+    useAutofetcher(fetchData, [fetchData], autorefresh);
+
+    const renderDescribeJson = () => {
```

Comment:
It's overhead to define this function, it only returns component without any additional logic.